File: Codeforces/379a.cc
/*
	Codeforces GoodBye 2013
	http://codeforces.com/contest/379/problem/A
	Guilherme Leobas
*/

#include <iostream>
#include <string>
#include <vector>
#include <set>
#include <map>
#include <deque>
#include <cstdio>
#include <cstdlib>
#include <algorithm>
#include <stack>
#include <queue>
#include <cmath>
#include <fstream>
#include <functional>
#include <iomanip>

using namespace std;

#define PI acos(-1)
#define cnt PI/180
#define INF 99999999
typedef vector<vector<pair<int, int> > > graph;

int main (){

	ios::sync_with_stdio(false);
	
	int a, b, resto;
	int result;
	resto = 0;

	cin >> a >> b;

	// result = a;
	result = 0;

	while (a > 0){
		a--;
		result += 1;
		resto++;
		if (resto >= b){
			a++;
			resto = 0;
		}
	}

	cout << result << endl;

	return 0;
}

File: Codeforces/379a.py
# Codeforces New Year 2013 Contest;
# http://codeforces.com/contest/379/problem/A

import sys;

def main ():
	s = raw_input ();
	s = s.split (' ');
	resto = 0;
	result = 0;
	a = int(s[0])
	b = int(s[1])


	while (a > 0):
		a -=1
		resto += 1
		result += 1;

		if (resto >= b):
			resto -= b;
			a += 1;


	print result;

main ();